tools/tasks/seed/build.assets.prod.ts

```diff
@@ -29,5 +31,6 @@ export = () => {
     '!' + join(ASSETS_SRC, '**', '*.js')
   ].concat(TEMP_FILES.map((p) => { return '!' + p; })))
     .pipe(onlyDirs(es))
+    .pipe(gulpIf((file) => ENABLE_JSON_COMMENT_STRIPPING && file.path.match(/.*.json$/), strip()))
```

Comment:
This approach is quite specific to .json files, if you would prefer I could make the configuration a whitelist of file extensions - did not see any other examples like this in the existing config options though so kept it more specific.